projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -452,6 +452,19 @@ public Optional<JobDeploymentStatus> readCronJob(String cronJobName) {
     return jobStatus.isPresent() ? jobStatus : readV1CronJob(cronJobName);
   }
 
```

Comment:
Can you add into the PR description, what was the actual cause of that error message, and how is that problem addressed to not print it? You bring 5 main/java/ class changes with this PR, and the cause and fix remain unknown.

Reply:
updated PR desc: 

```
The error happens when we deploy the job for the first time.
We check if the cronjob exists if it doesn't we build the image. 
This is a totally standard flow of operations and so shouldn't result in errors at the warn level. 
```